docs/content/docs/user-docs/irsa.md

```diff
@@ -67,6 +67,12 @@ For detailed instructions, refer to Amazon EKS documentation on how to [create a
 
 ## Step 2. Create an IAM role and policy for your service account
 
+{{% hint type="info" title="Note" %}}
+If you're trying to set up IRSA on OpenShift replace the `OIDC_PROVIDER` line in the bash sprint below with below command
+
+`OIDC_PROVIDER=$(oc get authentication cluster -ojson | jq -r .spec.serviceAccountIssuer | sed -e "s/^https:\/\///")`
```

Comment:
NIT: make this a code block instead of inline (i.e. "```" instead of "`")

Reply:
I have this as a `shell` code block to begin with, but I didn't love how it rendered inside of a `hint`. However, looking again it gives the `copy` option, which is probably better.